src/compiler/types.ts

```diff
@@ -1861,6 +1861,53 @@ namespace ts {
         getJsxIntrinsicTagNames(): Symbol[];
         isOptionalParameter(node: ParameterDeclaration): boolean;
 
+        /**
+         * Two types are identical iff they are references to the exact same fully qualified, resolved type (usually an alias to ===)
+         */
+        isIdenticalTo(a: Type, b: Type): boolean;
+        /**
+         * A type is a subtype of another if the collection of its members is contained within the collection of the other's members
+         */
+        isSubtypeOf(a: Type, b: Type): boolean;
+        /**
+         * Same as subtype, but all types are assignable to `any` and `any` is assignable to all types

```

Comment:
I can't view the spec inline in vscode. :3

I'll add a link to [this](https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md#3.11.2), [this](https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md#3.11.4) and [this](https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md#3.11.3), to the description of each but I think a succinct abbreviation inline would be useful (maybe highlighting where the relationship is formally used?).